### DISCUSSION TITLE
Timeline Graph in Hunt Not Showing Same As Time Span

### DISCUSSION THREAD

**Discussion by anitatata** [Security-Onion-Solutions/securityonion#8538] [Unsupported Versions] [answered]
Hello,

So in my security onion, the log is showing.
But when I select time span, examples I want to choose time span from August 1st 2022 until August 15th 2022.
The timeline graph is only showing from August 5th 2022 until August 15th 2022.

Below the capture:
![image](https://user-images.githubusercontent.com/13854947/184795235-a2579a1a-c819-45b0-9046-8ad9150ee62c.png)

Please kindly help me, is there any config I can change to showing the graph timeline properly?

Thank you

**Comment by dougburks** [comment]
Have you verified that you actually have data going back to August 1st?

**Comment by anitatata** [comment]
Thanks for your reply, where can I see the data log in the past? 
A weeks ago, I also check my security onion and the graph is showing from 2th August.

And today, the timeline graph is showing only for 7th August 2022 until today. I think it's only showing for 11 days in timeline.
Below capture:
![image](https://user-images.githubusercontent.com/13854947/185309122-6a582751-643e-45a4-8748-d31ef67903e5.png)

Could you please help me, where can I set the configuration? So it's not only showing for 11 days?

Thank you

**Comment by dougburks** [comment, marked as answer]
What is the output of the following?
```
df -h
sudo grep -r log_size_limit /opt/so/saltstack/local/pillar/
sudo so-elasticsearch-indices-list
```

**Reply by anitatata** [reply]
Thanks for your reply,

Here for df -h
![image](https://user-images.githubusercontent.com/13854947/185395035-c5e3e558-f467-4bb9-b6ef-6a74930c7a27.png)

and this for sudo so-elasticsearch-indices-list
![image](https://user-images.githubusercontent.com/13854947/185395205-742d3d3c-e95b-4aae-8583-4f8d96eda335.png)
![image](https://user-images.githubusercontent.com/13854947/185395231-920c9a9a-6034-4350-8f95-658269ee3c1e.png)
![image](https://user-images.githubusercontent.com/13854947/185395290-44c310cd-b343-488d-b282-6cb82d43beef.png)


Thank you

**Reply by dougburks** [reply]
Looking at your screenshots, your `/nsm` partition is only 176GB and so it is purging old data. You will most likely want much more than 176GB for a production deployment. For more information, please see:
https://docs.securityonion.net/en/2.3/hardware.html

**Reply by anitatata** [reply]
Hello

Thank you, currently my server already has a larger disk capacity.
![image](https://user-images.githubusercontent.com/13854947/187123127-44b4a0ad-26b4-4428-a2bb-ce041c18ca77.png)

On August 26, 2022 I took a log capture from August 1, 2022 to August 26, 2022 and it looks like there are logs from August 18 to August 26.
![image](https://user-images.githubusercontent.com/13854947/187123246-b9783019-28ac-4064-8f90-985ce42c5535.png)

Then today I took the capture log again from August 1, 2022 to August 26, 2022, but the logs that appear are only from August 22 to August 29.
![image](https://user-images.githubusercontent.com/13854947/187123346-77004847-0641-44eb-a087-721849d2f50e.png)

Do you have any other suggestions? So I can see logs over 1 week?

*additional info for question the output of this:
grep -r log_size_limit /opt/so/saltstack/local/pillar/

![image](https://user-images.githubusercontent.com/13854947/187123887-7132150e-3750-47aa-b7dc-b393634605d2.png)


Thank you

**Reply by dougburks** [reply]
Now that you have more disk space, you will want to increase your `log_size_limit` setting:
https://docs.securityonion.net/en/2.3/elasticsearch.html#deleting-indices
